posts/demos/_posts/2020-06-20-build-minimap.md

```diff
@@ -14,7 +14,7 @@ We will add on the minimap a rect that will be our indicator of the viewport.
 
 ### Setting up the idea
 
-We are going to create to canvases, on purposes both with a different aspect ratio and both different from the design aspect ratio.
+We are going to create two canvases with different aspect and design ratios.
```

Comment:
this sentence does not capture the exact message.
The 2 canvases have different aspect ratios. Both of them are also different from the content aspect ratio.
Do you understand what i mean?

you are displaying something that would fit in a square box for example, the main canvas can be 4:3 and the minimap could be 16:9. to generalize as much as possible

Reply:
Yes, maybe then just remove 'on purposes' and fix the 'to -> two' typo? 
'We are going to create two canvases, both with a different aspect ratio and both different from the design aspect ratio.'

Reply:
that works for me